src/typescript/07-functions/functions.md

```diff
@@ -20,135 +20,58 @@ In this part, we will:
 
 In TypeScript, we’re able to annotate function parameters to better guard our code. If the following, `add` is called  with two parameters that are not numbers TypeScript’s compiler will throw an error when compiled.
 
-```typescript
-function add(x: number, y: number): number {
-  return x + y;
-}
-
-add(1, 'three');
-//Argument of type 'string' is not assignable to parameter of type 'number'.
-```
+@sourceref ./7-1-1-parameters.ts
```

Comment:
Since you’re adding these files, feel free to rename them (the numbers don’t make sense).